test/helpers/protect-intrinsics-helpers.js

```diff
@@ -11,14 +11,17 @@ module.exports = {
  * @returns {Set<string>} All global intrinsic property names.
  */
 function getGlobalProperties() {
+  const comp = new Compartment().globalThis;
```

Comment:
```suggestion
  const compartmentGlobalThis = new Compartment().globalThis;
```
naming things is hard. this is only slightly better than `comp`

Reply:
oh this is just for a test helper? its fine